.kitchen.vagrant.yml

```diff
@@ -42,6 +42,9 @@ platforms:
 - name: debian-9
   driver_config:
     box: bento/debian-9
+- name: debian-10
+  driver_config:
+    box: docker-debian10-ansible
```

Comment:
This should probably be `bento/debian-10`

Reply:
Sweet! For context, I went with that image based on your link in https://github.com/dev-sec/ansible-ssh-hardening/issues/248#issue-502722819 but if bento's right, that sounds great to me!

Reply:
Bento images are for vagrant, for docker we use my own's.

Reply:
AH duh, sorry, makes sense!